### PR TITLE
Feature/more tests

### PR DESCRIPTION
**Pull request by hroest**

### CODE REVIEW

**Comment by hroest** [comment]
not sure what the problem was here, I restarted now all compile jobs

File: src/tests/class_tests/openms/source/MRMRTNormalizer_test.cpp [hunk 1]
```diff
@@ -49,13 +49,47 @@ START_TEST(MRMRTNormalizer, "$Id$")
 /////////////////////////////////////////////////////////////
 /////////////////////////////////////////////////////////////
 
+class MRMRTNormalizer_test : public MRMRTNormalizer
+{
+  public :
+
+    static int jackknifeOutlierCandidate(std::vector<double> & x, std::vector<double> & y)
+    {
+      return MRMRTNormalizer::jackknifeOutlierCandidate(x, y);
```
Comment (timosachsenberg) [inline src/tests/class_tests/openms/source/MRMRTNormalizer_test.cpp]: shouldn't these methods have _ as suffix?

**Comment by timosachsenberg** [comment]
merge or change the suffix?

**Comment by hroest** [comment]
added a few more things and move RANSAC code out to MATH/MISC

@grosenberger can you have a look at this? It should be more modular now and the RANSAC is re-useable

**Comment by grosenberger** [comment]
Looks good to me, thank you for improving the structure!

**Pull request merged**